src/daemon/org.projectatomic.rpmostree1.xml

```diff
@@ -92,6 +92,9 @@
 
          Available options:
          "reboot" (type 'b')
+         "install-packages" (type 'as')
+         "uninstall-packages" (type 'as')
+         "install-local-packages" (type 'ah')
```

Comment:
I'd initially been thinking that rather than add options to each existing DBus API method, we use this opportunity to add a *new* `CreateTransaction` (or whatever).  To ensure we're still testing all code paths, we'd have e.g. the `deploy` command still call the `Deploy` DBus API if no `--install` or `--remove` cmdline arguments were provided.

But if they are, then we'd call the new DBus API.  This might make it simpler to share code in the command line tools too.  Something like "scan argv for `--install`/`--remove`, and if found rewrite the command line to call into new txn cmdline code"?